docs/src/timearray.md

```diff
@@ -70,3 +70,9 @@ values
 colnames
 meta
 ```
+
+## Constructors
```

Comment:
I wonder if doc for TimeArray constructors shouldn't be written before "Fields getter functions"

Maybe other functions which have a TimeArray as parameter should all be listed here (or in a separated page)

like http://juliadata.github.io/DataFrames.jl/latest/lib/functions.html

Reply:
> I wonder if doc for TimeArray constructors shouldn't be written before "Fields getter functions"

okay, I can reagrange the sections.

>Maybe other functions which have a TimeArray as parameter should all be listed here (or in a separated page)

I think you want an API referece list, this is another issue.